README.md

```diff
@@ -133,6 +133,49 @@ sublime-phpcs is insanely powerful, but if you'd prefer automatic linting, [Subl
 
 Please see “[Setting up PHP CodeSniffer in Visual Studio Code](https://tommcfarlin.com/php-codesniffer-in-visual-studio-code/)”, a tutorial by Tom McFarlin.
 
+
+## Running your code through WPCS automatically using CI tools
+
+### [Travis CI](https://travis-ci.org/)
+
+To integrate PHPCS with WPCS with Travis, you'll need to install both `before_script` (or at `install`) and add the run command to the `script`.
+If your project uses Composer, the typical instructions might be different.
+
+If you use a matrix setup in Travis to test your code against different PHP and/or WordPress versions, you don't need to run PHPCS on each variant of the matrix as the results will be same.
+You can set an environment variable in the Travis matrix to only run the sniffs against one setup in the matrix.
+
+#### Travis CI example
+```bash
+language: php
+
+matrix:
+    include:
+        # Arbitrary PHP version to run the sniffs against.
+        - php: '7.0'
+          env: SNIFF=1
```

Comment:
p.s. @jrfnl YAML is a subset JSON spec and need to use spaces not tabs for indentation, I think most parsers will detect tabs if used but spaces will always be safer 😬 

See also https://core.trac.wordpress.org/browser/trunk/.editorconfig#L16